ampeg/src/test/java/DriverManagerFactory.java

```diff
@@ -0,0 +1,21 @@
+
+public class DriverManagerFactory {
+	
+	public static WebDriverManager getManager(String browser) {
+		
+		WebDriverManager webDriver;
+		
+		//defaults to Google Chrome
```

Comment:
Should thrown an exception as the default.
Remove browser.contains( ). This forces the test author to know exactly which browser he is calling for.